frontend/components/draft/sidebar.tsx

```
@@ -35,13 +42,28 @@ import { useLessonOrder } from "./metadata/hooks/useLessonOrder";
 import { Button } from "../ui/button";
 import { createDropletAnnouncement } from "@/lib/requests/feed";
 
+import { ContentActionButton } from "./metadata/content-action-button";
+
 export function Sidebar({
   user,
   droplet,
   authorizedUser,
```

ESLint

[error] 'authorizedUser' is defined but never used.

(@typescript-eslint/no-unused-vars)